brayns/Brayns.cpp

```diff
@@ -138,15 +138,14 @@ struct Brayns::Impl
         if(!geometryParameters.getXYZBFile().empty())
             _loadXYZBFile();
 
-        if(!volumeParameters.getFilename().empty())
+        if(!volumeParameters.getFilename().empty() || !volumeParameters.getFolder().empty())
         {
-            const Vector3ui& volumeDimensions = scene->getVolumeHandler()->getDimensions();
-            const Vector3f& volumeScale = volumeParameters.getScale();
+            const Vector3ui& volumeDimensions = scene->getVolumeHandler()->getDimensions( 0.f );
```

Comment:
TBH, I had the same feeling. But having to set the timestamp first before accessing the class members sort of makes the volumeHandler transactional, which is not great either. But I also agree that the API looks ugly with the timestamp for every method. So well, I changed the code as you suggested, just because it's you ;-) And it does look better now.